fake_filesystem_test.py

```diff
@@ -1400,10 +1400,11 @@ def testUtimeSetsCurrentTimeIfArgsIsNone(self):
 
   def testUtimeSetsCurrentTimeIfArgsIsNoneWithFloats(self):
     # set up
-    # time.time can report back floats, but it should be converted to ints
-    # since atime/ctime/mtime are all defined as seconds since epoch.
+    # we set os.stat_float_times() to False, so atime/ctime/mtime
+    # are converted as ints (seconds since epoch)
     time.time = _GetDummyTime(200.0123, 20)
     path = '/some_file'
+    fake_filesystem.FakeOsModule.stat_float_times(False)
     self._CreateTestFile(path)
```

Comment:
Please investigate and test the proper behavior when you create a file first, and then set `stat_float_times(False)`